packages/elvis-dropdown-lit/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@elvia/elvis-dropdown-lit",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "type": "module",
+  "files": [
+    "dist",
+    "react.ts"
+  ],
+  "scripts": {
+    "build": "tsup src/index.ts --dts --format esm --out-dir dist",
+    "start": "tsup src/index.ts --sourcemap --dts --format esm --out-dir dist --watch"
+  },
+  "dependencies": {
+    "@elvia/elvis-assets-icons": "^3.10.0",
+    "@elvia/elvis-colors": "^4.1.0",
+    "@elvia/elvis-typography": "^2.7.1",
+    "@lit/context": "^1.1.1",
+    "@lit/react": "^1.0.5",
+    "lit": "^3.1.3",
+    "lit-html": "^3.1.3"
+  },
+  "devDependencies": {
+    "tsup": "^8.0.2",
+    "typescript": "~5.4.5"
+  }
```

Comment:
Should we have `react` as an optional dependency? Considering we do an import from `react` in our `react.ts` file